Makefile

```diff
@@ -240,19 +242,27 @@ BUILD_DIR ?= $(PWD)/dist/rpmbuild
 SPEC_FILE ?= ${NAME}.spec
 SOURCE_NAME ?= ${NAME}-${VERSION}
 SOURCE_PATH := ${BUILD_DIR}/SOURCES/${SOURCE_NAME}.tar.bz2
+UNTRACKED_FILES := $(shell git ls-files --others --exclude-standard)
+TMP_SOURCE_DIR := $(shell rm -rf source-tmp.* && mktemp -d source-tmp.XXXX)
 
 rpm: rpm_prepare rpm_package_source rpm_build_source rpm_build
 
 rpm_prepare:
 	rm -rf $(BUILD_DIR)
 	mkdir -p $(BUILD_DIR)/SPECS $(BUILD_DIR)/SOURCES
 	cp $(SPEC_FILE) $(BUILD_DIR)/SPECS/
+	rsync -av --exclude=$(TMP_SOURCE_DIR) ./ $(TMP_SOURCE_DIR)/
```

Comment:
Use `cp -pr` instead. `rsync` is nice, but it may not always be available if our build environments change.